src/styles/_catalog.scss

```diff
@@ -1,8 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-items: center;
+  gap: 48px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 1fr);
+    max-width: 944px - 400 - 48 * 2;
```

Comment:
You can use fixed grid's colum's width, and don't write fixed max-width. Like this:
  @media (min-width: 488px) {
    grid-template-columns: repeat(2, 200px);
  }

Reply:
agree, you don't need `max-width`, remove it everywhere

Reply:
thx guys. i have fixed it)